### PR TITLE
Add Support to Minio DB

### PR DESCRIPTION
**Pull request by jsiloto**

### CODE REVIEW

File: ImageManager/ImageManager.py [hunk 1]
```diff
@@ -40,10 +52,25 @@ def get_image(imageid):
             return format_response(e.error_code, e.message)
 
 
+@image.route('/image/<imageid>/binary', methods=['GET'])
+def get_image_binary(imageid):
+    try:
+        init_tenant_context(request, db, minioClient)
+        orm_image = assert_image_exists(imageid)
+        filename = imageid + '.hex'
+        return send_from_directory(directory='/tmp/', filename=filename)
```
Comment (mmagr) [inline ImageManager/ImageManager.py]: Here we're actually returning the "cache" copy used when the user uploads to minio?

File: ImageManager/ImageManager.py [hunk 2]
```diff
@@ -94,21 +119,29 @@ def create_image():
...
         orm_image = assert_image_exists(imageid)
         if orm_image.confirmed:
             raise HTTPRequestError(400, "Binary already exists")
 
         orm_image.confirmed = True
         file_data = parse_form_payload(request)
-
+        extension = file_data.filename.rsplit('.', 1)[1].lower()
+        filename = imageid + '.' + extension
+        file_data.save(os.path.join('/tmp/', filename))
```
Comment (mmagr) [inline ImageManager/ImageManager.py:135]: is this what we're serving through `GET /images/:id/binary`?

File: ImageManager/ImageManager.py (hunk 1 above)
Comment (jsiloto) [reply ImageManager/ImageManager.py]: Fixed

File: ImageManager/ImageManager.py [hunk 3]
```diff
@@ -1,35 +1,50 @@
...
 LOGGER.addHandler(logging.StreamHandler())
 LOGGER.setLevel(logging.DEBUG)
 
 
+@image.route('/image/', methods=['GET'])
+def get_all():
+    try:
+        init_tenant_context(request, db, minioClient)
+        images = get_all_images()
+        json_images = [image_schema.dump(i).data for i in images]
```
Comment (giovannicuriel) [inline ImageManager/ImageManager.py]: I had a few problems with dump(x).data with the new Marshmallow version - apparently, dump() returns the dict directly (without `data` attribute)

Comment (jsiloto) [reply ImageManager/ImageManager.py]: Yeah, I missed that.
Fixed

**Review by mmagr — APPROVED** [review]

**Pull request merged**